src/features/hotspots/transfers/HotspotTransfer.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from 'react'
+import { StyleSheet, TextInput } from 'react-native'
+import { Hotspot } from '@helium/http'
+import animalName from 'angry-purple-tiger'
+import { useTranslation } from 'react-i18next'
+import { useNavigation } from '@react-navigation/native'
+import Text from '../../../components/Text'
+import TouchableOpacityBox from '../../../components/TouchableOpacityBox'
+import CloseModal from '../../../assets/images/closeModal.svg'
+import TransferHotspotIcon from '../../../assets/images/transferHotspotIcon.svg'
+import Box from '../../../components/Box'
+import Button from '../../../components/Button'
+
+type Props = {
+  onCloseTransfer: () => void
+  onCloseSettings: () => void
+  hotspot: Hotspot
+}
+
+const HotspotTransfer = ({
+  onCloseTransfer,
+  onCloseSettings,
+  hotspot,
+}: Props) => {
+  const hotspotName = animalName(hotspot.address)
+  const { t } = useTranslation()
+  const navigation = useNavigation()
+  const [typedName, setTypedName] = useState('')
+
+  const handleTypeName = (text: string) => {
+    setTypedName(text)
+  }
+
+  const typedHotspotName = () => {
```

Comment:
i was confusing this with typescript types in my head for a moment. of course i can't immediately think of a better name off the top of my head 😛

Reply:
Haha hmm, I could make it `enteredHotspotName`? Its the name they are typing to confirm the transfer